src/osversion/VersionTranslator.cpp

```diff
@@ -8310,6 +8306,24 @@ namespace osversion {
         m_refactored.push_back(RefactoredObjectData(object, newObject));
         ss << newObject;
 
+      } else if ((iddname == "OS:Coil:Heating:Gas") || (iddname == "OS:Coil:Heating:Gas:MultiStage")
+                 || (iddname == "OS:Coil:Heating:Gas:MultiStage:StageData") || (iddname == "OS:Coil:Heating:Desuperheater")) {
+
+        // Change Parasitic Electric Load to On Cycle Parasitic Electric Load
+        // Change Parasitic Gas Load to Off Cycle Parasitic Gas Load
+
+        auto iddObject = idd_3_7_0.getObject(iddname);
+        IdfObject newObject(iddObject.get());
+
+        for (size_t i = 0; i < object.numFields(); ++i) {
+          if ((value = object.getString(i))) {
+            newObject.setString(i, value.get());
+          }
+        }
+
+        m_refactored.push_back(RefactoredObjectData(object, newObject));
+        ss << newObject;
```

Comment:
I don't think you need VT at all actually. The VT Test, good, we want to make sure.

But this is nothing but a name change, the fields stay the same. Try removing VT and check the test.

Reply:
I thought about that... but if we weren't changing them in the cpp get/set methods, so then how would the field names be changing because of default vt?

Reply:
I tried commenting out the new vt. And all tests seems to pass. So I'm pretty confused - default vt is or isn't changing the names of the fields?

Reply:
Alright, so, if you do nothing, the object is serialized as is.

Then the stringstream is reloaded with the 3.7.0 IDD. The IddObject is just gotten by its name, and the field values go to the idf object (no issue there),. The IddObject is the the bit that carries the field names (and field properties). And that's gotten from the new idd, so the field names are "updated".

Let me know if this isn't clear (I fear it isn't, but I'm aiming for quickness here).

Reply:
So, before, model tests were passing (even before changing in model cpp from, e.g., `OS_Coil_Heating_GasFields::ParasiticElectricLoad` to `OS_Coil_Heating_GasFields::OnCycleParasiticElectricLoad`) because was using old idd. Now it's using new idd, and vt serializes to it by default.